src/components/StatePicker/StatePicker.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import { getBirds } from '../../apiCalls';
```

Comment:
It looks like we don't need this line because we don't use "getBirds" anymore in the code

Reply:
Good catch! I'll remove this 👍